src/components/persons.js

```diff
@@ -3417,5 +3417,21 @@ export const persons = [
       state: "MS",
       country: "Brazil"
     }
```

Comment:
{
    id: 207,
    name: "Dimitris Kallimanis",
    img:
      "https://avatars1.githubusercontent.com/u/17277637?s=400&u=7732d8e125e4f5985b7e27aa75d8edcae85c5931&v=4",
    links: {
      website: "https://www.dimitrios-kallimanis.com",
      linkedin: "://www.linkedin.com/in/dimitrios-kallimanis-b0a27a62/",
      github: "://github.com/kallimanis79"
    },
    jobTitle: "Front-End Developer/Web Developer/Full Stack Developer",
    location: {
      city: "Leuven",
      state: "Brussels",
      country: "Belgium"
    }
 },
];

Reply:
Add me in JobsBoard. Thank you.